framework/src/Volo.Abp.Ddd.Domain/Volo/Abp/DependencyInjection/DbContextTypeWithTenandside.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using Volo.Abp.MultiTenancy;
+
+namespace Volo.Abp.DependencyInjection;
+
+public class MultiTenantDbContextType
```

Comment:
the file name `DbContextTypeWithTenandside.cs` has misspell it should be `DbContextTypeWithTenantSide.cs`.
and it is better to rename file name to match the class name.